GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.References.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GitFyle.Core.Api.Models.Foundations.Contributions;
+using Microsoft.EntityFrameworkCore;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void SetContributionReferences(ModelBuilder modelBuilder)
+        {
+            int maxStringLength = 255;
```

Comment:
Do not make use of a variable to set the max length.  If you use the variable more than once it will lead to entanglement and you can unintendedly change other properties if you alter the value.  Rather do `.HasMaxLength(255)`